test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -286,9 +286,9 @@ public static TheoryData<HtmlEncoder> HtmlEncoderData
             // Arrange
             var page = CreatePage(v =>
             {
-                v.Path = "/Views/TestPath/Test.cshtml";
+                v.Path = "/Views/Shared/_Layout.cshtml";
                 v.RenderSection("bar");
-            });
+            }, context: CreateViewContext(viewPath: "/Views/TestPath/Test.cshtml"));
```

Comment:
Yeah this seems like exactly the test that needs to be updated... the name of this test method couldn't be any better...